src/pages/components/image.mdx

```diff
@@ -14,11 +14,80 @@ import ComponentFooter from "components/ComponentFooter";
 
 </InlineNotification>
 
-<AnchorLinks>
 
+<AnchorLinks>
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Content guidance</AnchorLink>
 <AnchorLink>Development documentation</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
-<ComponentFooter name="Image" type="ui" />
+## Overview
+
+Image at its core is a wrapper that adds an image to the page. You can pass multiple images to a particular image component to render a different image at various breakpoints - this is useful if you need to render different images at different screensizes. 
+
+We recommend displaying images at 1x1, 2x1, 4x3, and 16x9 aspect ratios. For guidance on choosing or creating an IBM image, see the <a href="https://www.ibm.com/design/language/photography/overview" target="_blank">photography guidelines</a> on the IBM Design Language site. 
+
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Image of Image default in context](../../images/component/image/image-overview-default.jpg)
+
+</Column>
+</Row>
+
+## Options
+
+### Caption and zoom
+
+An image can include a caption beneath the image itself to provide more context. With the caption feature, the image can also be clicked to open an overlay (Lightbox media viewer) with a larger version of the image. 
+
+Inside the viewer, the caption becomes the heading and there is an opportunity for additional body text.
+
+For more about Image with caption, see the <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Image-with-caption" target="_blank">functional specs</a>.
+
+<Row>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: single selection with no selected filter.](../../images/component/image/image-options-caption-hover.jpg)
+
+<Caption>The image with a caption displayed in context</Caption>
+
+</Column>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: single selection with selected filter.](../../images/component/image/image-options-caption-lightbox.jpg)
+
+<Caption>Once clicked, the Lightbox media viewer displays a larger version of the image</Caption>
+
+</Column>
+
+</Row>
+
+### Border
+
+An image can include a 1px #E0E0E0 ($decorative-01) border to help frame the images better when the page background and image background colors are the same.
+
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Image of Image default in context](../../images/component/image/image-options-border.png)
+
+</Column>
+</Row>
+<Caption>The border separates the image from the background with the same color</Caption>
+
+## Content guidance
```

Comment:
@RichKummer wondering if we get Jonathan to work on this table to get the updated version for this PR. Question for @shixiedesign. It probably wouldn't take him too long.